runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigurationManager.java

```diff
@@ -292,28 +244,49 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
 
                 tuning.affinity(binding.id, affinity);
             }
-
-            CompletableFuture<Void> future = CompletableFuture.completedFuture(null);
-            for (DispatchAgent dispatcher : dispatchers)
-            {
-                future = CompletableFuture.allOf(future, dispatcher.attach(namespace));
-            }
-            future.join();
-
-            extensions.forEach(e -> e.onConfigured(context));
         }
         catch (Throwable ex)
         {
-            errorHandler.onError(ex);
+            logError(ex.getMessage());
         }
 
         if (!errors.isEmpty())
         {
-            errors.forEach(msg -> errorHandler.onError(new JsonException(msg)));
+            errors.forEach(this::logError);
         }
+        return namespace;
+    }
 
-        // TODO: repeat to detect and apply changes
+    private void logError(
+        String message)
+    {
+        logger.accept("Configuration parsing error: " + message);
+    }
```

Comment:
Please move this private method to the bottom of the class, after the public methods.